src/piggui.rs

```diff
@@ -452,17 +473,26 @@ impl Application for Gpio {
 // Hardware Configuration Display
 fn hardware_view(hardware_description: &HardwareDescriptor) -> Element<'static, Message> {
     let hardware_info = Column::new()
-        .push(Text::new(format!("Hardware: {}", hardware_description.hardware)).size(20))
-        .push(Text::new(format!("Revision: {}", hardware_description.revision)).size(20))
-        .push(Text::new(format!("Serial: {}", hardware_description.serial)).size(20))
-        .push(Text::new(format!("Model: {}", hardware_description.model)).size(20))
+        .push(Text::new(format!(
```

Comment:
formatting again. The original should be under 80 columns and good I think, but I will check rustfmt